service.go

```diff
@@ -81,6 +85,10 @@ func (ts *telemetryService) ServeUI(ctx context.Context, filters TelemetryFilter
 	if err != nil {
 		return nil, err
 	}
+
+	filterCountries := unfilteredSummary.Countries
```

Comment:
no need to declare these here. You can just place them directly into the data struct

`FilterCountries: unfilteredSummary.Countries`
`FilteredCities: unfilteredSummary.Cities`